client/game.hpp

```diff
@@ -1,4 +1,5 @@
-#pragma once
```

Comment:
I have updated the style guides to indicate using `#prgram once`. You could revert some of the changes in this PR, or it can be part of a bigger style/format PR.